packages/generator/src/vdm-types.ts

```diff
@@ -89,6 +90,7 @@ export interface VdmComplexType {
   typeName: string;
   factoryName: string;
   fieldType: string;
+  namespace: string;
```

Comment:
I'm sure there is a reason, but why does the `VdmComplexType` have a namespace. In `EdmxComplexType` it's obvious

Reply:
I add namespace info to `VdmEntity` as well, because both of them are used for matching the return type of the action/function import. This seems to be a temporary solution, I guess we will get rid of it when introducing additional abstraction.